Header header
Header image_header
int32 num_humans
Human[] human_list